built/History.js

```diff
@@ -4,22 +4,22 @@ var __extends = (this && this.__extends) || function (d, b) {
     function __() { this.constructor = d; }
     d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
 };
-var React = require('react');
```

Comment:
Not related to your code, but to your setup: There seems to be a nuance of difference in your build. Would be nice that all contributors do not keep overwriting each other's build files due to these differences. Perhaps in the future PRs should only include source changes so that churn is minimized.